i18n/it/cachyos_hello.ftl

```diff
@@ -0,0 +1,66 @@
+# About dialog
+about-dialog-title = CachyOS Hello
+about-dialog-comments = Schermata di benvenuto per CachyOS
+
+# Tweaks page
+tweaks = Modifiche
+fixes = Correzioni
+applications = Applicazioni
+removed-db-lock = Il blocco della base di dati di pacman è stato rimosso!
+lock-doesnt-exist = Il blocco della base di dati di pacman non esiste!
+orphans-not-found = Nessun pacchetto orfano trovato!
+package-not-installed = Il pacchetto '{$package_name}' non è stato installato!
+
+# Dns Connections page
+dns-settings = Impostazioni DNS
+select-connection = Seleziona Connessione:
+select-dns-server = Seleziona server DNS:
+apply = Applica
+reset = Reimposta
+dns-server-changed = Il server DNS è stato cambiato con successo!
+dns-server-failed = Impostazione del server DNS non riuscita!
+dns-server-reset = Il server DNS è stato reimpostato!
+dns-server-reset-failed = Reimpostazione del server DNS non riuscita!
+
+# Tweaks page (tweaks)
+tweak-enabled-title = {$tweak} abilitato
+
+# Tweaks page (fixes)
+remove-lock-title = Rimuovi il blocco della base di dati
+reinstall-title = Reinstalla tutti i pacchetti
+refresh-keyrings-title = Ricarica i keyring
+update-system-title = Aggiorna il sistema
+remove-orphans-title = Rimuovi gli orfani
+clear-pkgcache-title = Pulisci la cache dei pacchetti
+rankmirrors-title = Classifica i mirror
+dnsserver-title = Cambia server DNS
+
+# Main Page (buttons)
+button-about-tooltip = Informazioni
+button-web-resource-tooltip = Risorse in rete
+button-development-label = Sviluppo
+button-software-label = Software
+button-donate-label = Dona
+button-forum-label = Forum
+button-installer-label = Lancia l'installer
+button-involved-label = Partecipa
+button-readme-label = Leggimi
+button-release-info-label = Informazioni di rilascio
+button-wiki-label = Wiki
+
+# Main Page (sections)
+section-docs = DOCUMENTAZIONE
+section-installer = INSTALLAZIONE
+section-support = SUPPORTO
+section-project = PROGETTO
+
+# Main Page (body)
+offline-error = Impossibile avviare l'installazione online! Connessione a internet assente
+tweaksbrowser-label = App/Modifiche
+appbrowser-label = Installa App
+launch-start-label = Lancia all'avvio
+welcome-title = Benvenuto in CachyOS!
+welcome-body =
+    Grazie per esserti unito alla nostra comunità!
+
+    Noi, gli sviluppatori di CachyOS, speriamo che ti divertirai a usare CachyOS tanto quanto noi ci divertiamo a costruirlo. I collegamenti qua sotto ti aiuteranno a fare i primi passi col tuo nuovo sistema operativo. Per cui goditi l'esperienza e non esitare a mandarci la tua opinione.
```

Comment:
much probably it can be more appropriate to write like that:

```
Grazie per unirti alla nostra communità!

Noi, gli sviluppatori di CachyOS, speriamo che tu trovi l'utilizzo di CachyOS altrettanto piacevole quanto lo troviamo noi nello svilupparlo. I link sottostanti ti aiuteranno a orientarti nel tuo nuovo sistema operativo. Goditi questa esperienza e non esitare a inviarci i tuoi feedback
```

Reply:
Hmm, I like most of the second part a lot, how about this:
```ftl
welcome-body =
    Grazie per esserti unito alla nostra comunità! # "Grazie per unirti" sounds very unnatural to me

    Noi, gli sviluppatori di CachyOS, speriamo che tu trovi l'utilizzo di CachyOS altrettanto piacevole quanto lo troviamo noi nello svilupparlo. I collegamenti sottostanti ti aiuteranno a orientarti nel tuo nuovo sistema operativo. Per cui goditi l'esperienza e non esitare a inviarci la tua opinione. # I'd like to avoid anglicisms where possible
```

Reply:
yeah is perfect that!